config.go

```diff
@@ -35,8 +35,8 @@ const (
 	defaultLogDirname            = "log"
 	defaultLogFilename           = "dcrpool.log"
 	defaultDBFilename            = "dcrpool.kv"
-	defaultTLSCertFilename       = "dcrpool.cert"
-	defaultTLSKeyFilename        = "dcrpool.key"
+	defaultGuiTLSCertFilename    = "dcrpool.cert"
+	defaultGuiTLSKeyFilename     = "dcrpool.key"
```

Comment:
We already have instances in the codebase where `GUI` is capitalized so update these for consistency.